brave/brave-core
# Added 'Manifest V2 extensions' section in settings.

Resolves https://github.com/brave/brave-browser/issues/28367

https://user-images.githubusercontent.com/73575789/216320363-1b325600-b7db-4aa7-a68c-dde06572445c.mp4


![image](https://user-images.githubusercontent.com/73575789/218736593-7d85bb04-b6c9-45a2-93fe-b6a9eabcd074.png)


## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Launch Brave with ```ExtensionsManifestV2``` feature enabled.
2. Go to brave://settings/extensions
3. Click to ```Manifest V2 Extensions``` section
4. You should see 4 extensions
5. Enable/Disable toggles will isntall/uninstall corresponding extensions
